fineract-core/src/main/java/org/apache/fineract/organisation/monetary/domain/Money.java

```diff
@@ -18,27 +18,21 @@
  */
 package org.apache.fineract.organisation.monetary.domain;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Embeddable;
 import java.math.BigDecimal;
+import java.math.MathContext;
 import java.math.RoundingMode;
 import java.util.Iterator;
+import lombok.Getter;
 import org.apache.fineract.organisation.monetary.data.CurrencyData;
 
-@Embeddable
```

Comment:
Isn't this going to be a breaking change? Is there any JPA entity using this class as an embeddable?

Reply:
Fixed.

Reply:
I dont think we are using it in any JPA entity, but worth to double check.

Reply:
I mean guys, I'm okay with removing the JPA annotations if it is not used anywhere.